content/index.html

```diff
@@ -82,15 +79,15 @@ <h3 class="learning__header-title">Start building a better web.</h3>
 
   <div class="join-us container container--lp">
     <div class="join-us__header">
-      <h2 class="overline join-us__header-overline no-link">Join Us</h2>
+      <h2 class="overline join-us__header-overline no-link">Save your progress</h2>
       <h3 class="join-us__header-title no-link">
-        Sign in to manage real-time audits and track your learning progress.
+        See how your website is performing over time, and keep track as you learn on your own schedule.
```

Comment:
Suggestion to make this a bit more active-- and it will also bring content onto two lines instead of three: 'See how your website performs over time, and keep track as you learn on your own schedule'.

Reply:
Nice refinement!